src/PrestaShopBundle/Install/Upgrade.php

```diff
@@ -250,10 +250,12 @@ private function defineConst()
             if (empty($this->oldVersion)) {
                 $this->oldVersion = Configuration::get('PS_INSTALL_VERSION');
             }
-            // fix : complete version number if there is not all 4 numbers
-            // for example replace 1.4.3 by 1.4.3.0
-            // consequences : file 1.4.3.0.sql will be skipped if oldversion = 1.4.3
-            // @since 1.4.4.0
+            /**
```

Comment:
Phpcs say you break something here :)

Reply:
Can the order be the issue, since syntax is `@since [version] [<description>]` ?